composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "drupal/commerce_bluesnap",
+  "type": "drupal-module",
+  "description": "Provides Drupal Commerce integration with the BlueSnap Payment Gateway.",
```

Comment:
**CHANGE REQUEST**

"Provides Drupal Commerce integration with the BlueSnap Payment Platform."